client/ionic/src/components/ShareButton.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from "react";
+import { IonContent, IonButton } from "@ionic/react";
+import { SocialSharing } from '@ionic-native/social-sharing';
+
+
+interface ShareConfig {
+    shareSTR: string;
```

Comment:
super nitpicky code style comment (which brings up the point - we should decide as a group how much we want to standardize and enforce), but I usually prefer camelcase naming (eg `shareStr`). I don't want to impose my own preference yet, but we should decide on standards...

Reply:
Wait for #82